src/parser/jobs/ast/modules/Overheal.tsx

```diff
@@ -1,111 +1,35 @@
-		const nonHotOverhealPercent = 100 * this.healingDirect / (this.healingDirect + this.overhealDirect)
-		const petOverhealPercent = 100 * this.healingPet / (this.healingPet + this.overhealPet)
-		const hotOverhealPercent = 100 * this.healingOverTime / (this.healingOverTime + this.overhealOverTime)
-		const sourcesSum = this.healingOverTime + this.healingDirect + this.healingPet
-		const sourcesSumOverheal =  this.overhealOverTime + this.overhealPet + this.overhealDirect
-
-		this.checklist.add(new TieredRule({
-			name: <Trans id="ast.overheal.rule.name">Avoid overheal</Trans>,
-			description: <><Trans id="ast.overheal.rule.description">
-				Avoid wasting heals by healing for more than required to fill a target's HP bar. While some overheal is inevitable, overheal only serves to generate more enmity, for no gain. Being efficient with healing additionally helps with your MP management.
-			</Trans>
-			<ul>
-				<li><Trans id="ast.overheal.requirement.nonhot"> Overheal (non-HoT): {this.invertPercent(nonHotOverhealPercent)}%</Trans></li>
-				{petOverhealPercent > 0 && <li><Trans id="ast.overheal.requirement.earthly-star"> Overheal (Earthly Star): {this.invertPercent(petOverhealPercent)}%</Trans></li>}
-				<li><Trans id="ast.overheal.requirement.hot"> Overheal (HoT): {this.invertPercent(hotOverhealPercent)}%</Trans></li>
-			</ul></>,
-			tiers: SEVERITY_TIERS,
-			requirements: [
-				new InvertedRequirement({
-					name: <Trans id="ast.overheal.requirement.all"> Overheal (all sources)</Trans>,
-					percent: (100 * sourcesSum / (sourcesSum + sourcesSumOverheal)), // put in inverted data
-				}),
+	protected override checklistRuleBreakout = true
+	protected override displayPieChart = true
+	protected override displaySuggestion = true
+
+	protected override trackedHealCategories = [
+		{
+			name: <Trans id="ast.overheal.hot.name">Healing over Time</Trans>,
+			color: SuggestedColors[1],
+			trackedHealIds: [
+				this.data.statuses.ASPECTED_HELIOS.id,
+				this.data.statuses.WHEEL_OF_FORTUNE_DIURNAL.id,
+				this.data.statuses.WHEEL_OF_FORTUNE_NOCTURNAL.id,
+				this.data.statuses.ASPECTED_BENEFIC.id,
+				this.data.statuses.DIURNAL_OPPOSITION.id,
+				this.data.statuses.DIURNAL_INTERSECTION.id,
+				this.data.statuses.DIURNAL_BALANCE.id,
 			],
-		}))
-	}
-	private invertPercent(value: number) {
-		const invertedValue = 100 - value
-		return invertedValue.toFixed(2)
-	}
-}
-
-// yeh, I'm not doing this in core, but I really want to show overheal as overheal, since that's what the community understands
-export class InvertedRequirement extends Requirement {
-	get percentInverted() {
-		return 100 - this.percent
-	}
-
-	override get content() {
-		if (this._percent !== null || this.value === null) { return `${this.percentInverted.toFixed(2)}%` }
-		return `${this.value.toFixed(0)}/${this.target.toFixed(0)}` // avoid weird floating point shit
-	}
+		},
+		{
+			name: <Trans id="ast.overheal.earthlystar.name">Earthly Star</Trans>,
```

Comment:
Does this really need translation over something like `<ActionLink showIcon=false action='EARTHLY_STAR'/>`? That way it's auto-localised, iirc can also meme `showTooltip=false` to just keep the name localised without having any mouseover. If it needs to be a React.JSX object, you can use a Fragment I think.

Reply:
Mostly did it this way to keep the current behavior of AST and match the structure of SCH/WHM. Though I couldn't keep the i18n key because it's a little different format now (no trailing colon) so I updated the context in the key.

Reply:
Fair enough.